applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebug.conf

```diff
@@ -66,7 +66,8 @@ CONFIG_HEAP_MEM_POOL_MIN_SIZE=32
 
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 
-CONFIG_SYS_PM_POLICY_APP=y
+CONFIG_PM_POLICY_APP=y
+CONFIG_PM_DEEP_SLEEP_STATES=y
```

Comment:
No need to select this explicitly. Power manager Kconfig option selects this one.

Reply:
Also for other configurations.